packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -35,7 +35,7 @@ export const BlockLoaderInput: React.VFC = () => {
     const normalizedUrl = createNormalizedBlockUrl(blockUrl);
 
     blockView.manager
-      .fetchAndDefineBlock(normalizedUrl)
+      .fetchAndDefineBlock(normalizedUrl, true)
```

Comment:
```suggestion
      .fetchAndDefineBlock(normalizedUrl, { bustCache: true })
```

(optional) Makes it easier to see what `true` is for.

Reply:
Makes sense! https://github.com/hashintel/hash/pull/572/commits/81bb2c57fa1c2eb99cda266952875588940849f7